ConfigData/phase GEMS Comparison.py

```diff
@@ -346,25 +405,82 @@ def GEMSPlot(AtPct, ProtosolarToSi):
         #     ChondriticInds.append(TickLabels.index(pb.ElementalSymbols[Zminus1+1]))
         #     ChondriticVals.append(ProtosolarToSi[Zminus1])
     # We will be plotting so clear the plot that may already be plotted.
-    plt.figure(1)
-    plt.clf()
+
+
+    # FIXME
+    # TODO Using mpld3 to convert matplotlib codes to HTML. 
+    # mpld3 has bugs of converting set_yscale('log'). I can use the ax.plot to solve that but the axhline and errorbar is not abel to be resolved.
+
+    # fig = Figure(figsize = (6,4.5))
+    # ax = fig.subplots()
+    # ax.plot(GEMSInds, GEMSVals, marker='o', color='red', markersize = 15, alpha=0.5, linestyle='', label=RefStr)
```

Comment:
If all this stuff is commented out that means it isn't being used.  It should be deleted.  Remember the old  code is still there in the commit history.